modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_navs.scss

```diff
@@ -0,0 +1,387 @@
+// nav-base-link
+
+$nav-base-link: (
+	after: (
+		bottom: 0,
+		content: none,
+		display: block,
+		position: absolute,
+		left: 0,
+		right: 0,
+		width: auto,
+	),
+);
+
+// Primary tab
+
+$nav-underline-link: (
+	active-class: (
+		background-color: var(--tab-primary-item-background-color-active),
+		border-color: var(--tab-primary-item-border-color-active),
+		box-shadow: var(--tab-primary-item-box-shadow-active),
+		color: var(--tab-primary-item-color-active),
+		font-weight: var(--tab-primary-item-font-weight-active),
+		after: (
+			background-color: var(--tab-primary-item-active-bar-color-active),
+			height: var(--tab-primary-item-active-bar-thickness),
+			content: '',
+		),
+		inline-item: (
+			color: var(--tab-primary-item-icon-color-active),
+		),
+	),
+	after: (
+		border-radius: var(--tab-primary-item-active-bar-border-radius),
+		height: var(--tab-primary-item-active-bar-thickness),
+		width: var(--tab-primary-horizontal-item-active-bar-width),
+	),
+	background-color: var(--tab-primary-item-background-color),
+	border-color: var(--tab-primary-item-border-color),
+	border-radius: var(--tab-primary-item-border-radius),
+	border-style: solid,
+	border-bottom-width: var(--tab-primary-horizontal-item-border-bottom-width),
+	border-left-width: var(--tab-primary-horizontal-item-border-left-width),
+	border-right-width: var(--tab-primary-horizontal-item-border-right-width),
+	border-top-width: var(--tab-primary-horizontal-item-border-top-width),
+	color: var(--tab-primary-item-color),
+	disabled: (
+		border: 0,
+		after: (
+			content: none,
+		),
+		background-color: var(--tab-primary-item-background-color-disabled),
+		color: var(--tab-primary-item-color-disabled),
+		font-weight: var(--tab-secondary-item-font-weight),
+		inline-item: (
+			color: var(--tab-primary-item-icon-color-disabled),
+		),
+	),
+	font-size: var(--tab-primary-item-font-size),
+	font-weight: var(--tab-primary-item-font-weight),
+	hover: (
+		background-color: var(--tab-primary-item-background-color-hover),
+		border-color: var(--tab-primary-item-border-color-hover),
+		color: var(--tab-primary-item-color-hover),
+		box-shadow: var(--tab-primary-item-box-shadow-hover),
+		font-weight: var(--tab-primary-item-font-weight-active),
+		after: (
+			background-color: var(--tab-primary-item-active-bar-color-hover),
+			content: '',
+		),
+		inline-item: (
+			color: var(--tab-primary-item-icon-color-hover),
+		),
+	),
+	line-height: var(--tab-primary-item-line-height),
+	inline-item: (
+		color: var(--tab-primary-item-icon-color),
+		font-size: var(--tab-primary-item-icon-font-size),
+		margin-y: var(--tab-primary-item-inline-item-spacer-y),
+	),
+	inline-item-after: (
+		margin-left: var(--tab-primary-item-inline-item-spacer-x),
+	),
+	inline-item-before: (
+		margin-right: var(--tab-primary-item-inline-item-spacer-x),
+	),
+	padding: var(--tab-primary-horizontal-item-padding-y)
+		var(--tab-primary-horizontal-item-padding-x),
+);
+
+$nav-vertical: (
```

Comment:
This should probably not be under Primary Tab, but under it's own Nav Vertical section

Reply:
Fixed